CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## 1.0.1
+
+* Bug Fix: Make sure compute_asset_path is always available ([Details](https://github.com/faucet-pipeline/faucet_pipeline_rails/pull/39))
+
+Thanks to Dirk for finding and fixing this bug.
+
+## 1.0.0
+
+* First stable release of faucet-pipeline
```

Comment:
… for Ruby on Rails? (Though it would be pretty badass to list faucet-spring releases here.)